.expeditor/config.yml

```diff
@@ -34,7 +34,14 @@ merge_actions:
   - built_in:build_gem:
       only_if:
         - built_in:bump_version
+  - built_in:publish_rubygems:
```

Comment:
Would love a review from @chef/expeditor-maintainers - because this gem is in beta and to limit the number of manual gates we need to perform to get Chef Apply updates into Chef Workstation, I want to push this gem to Rubygems on every merge

Reply:
This should be fine @tyler-ball !